core/executor/full_exit_nft_executor.go

```diff
@@ -246,5 +252,25 @@ func (e *FullExitNftExecutor) GenerateTxDetails() ([]*tx.TxDetail, error) {
 		CollectionNonce: exitAccount.CollectionNonce,
 	})
 
+	// create account empty delta
+	order++
+	accountOrder++
+	txDetails = append(txDetails, &tx.TxDetail{
+		AssetId:      types.EmptyAccountAssetId,
+		AssetType:    types.FungibleAssetType,
+		AccountIndex: txInfo.CreatorAccountIndex,
+		AccountName:  creatorAccount.AccountName,
+		Balance:      creatorAccount.AssetInfo[types.EmptyAccountAssetId].String(),
+		BalanceDelta: types.ConstructAccountAsset(
+			types.EmptyAccountAssetId,
+			types.ZeroBigInt,
+			types.ZeroBigInt,
+		).String(),
+		Order:           order,
+		AccountOrder:    accountOrder,
+		Nonce:           creatorAccount.Nonce,
+		CollectionNonce: creatorAccount.CollectionNonce,
+	})
+
```

Comment:
In this pr, you should update zkbnb-crypto version in go.mod, like this pr: https://github.com/bnb-chain/zkbnb/pull/241